vendor/github.com/sirupsen/logrus/go.mod

```diff
@@ -1,5 +1,7 @@
 module github.com/sirupsen/logrus
 
+go 1.14
```

Comment:
Any reason why this is changed?

Reply:
No idea why this is even picking up as a change since this was changed in April. 

<img width="643" alt="image" src="https://user-images.githubusercontent.com/8770192/92960718-196cd780-f43c-11ea-9fa5-f94b0ff9fc31.png">